doc/introduction.rst

```diff
@@ -83,28 +75,63 @@ Why is machine learning relevant to NeuroImaging: a few examples
     the data (e.g. in `[Thirion 2009]
     <http://www.springerlink.com/content/7377x70p5515v778/>`_).
 
-.. _installation:
+Glossary: machine learning vocabulary
+--------------------------------------
+
+:Supervised learning:
+
+    :ref:`supervised learning <decoding>` is interested in predicting an
+    **output variable**, or **target**, `y`, from **data** `X`.
+    Typically, we start from labeled data (the **training set**) for
```

Comment:
label -> for example different experimental conditions or healthy versus diseased participant data